src/wp-admin/includes/class-wp-upgrader.php

```diff
@@ -156,6 +156,7 @@ public function generic_strings() {
 		/* translators: %s: Directory name. */
 		$this->strings['fs_no_folder'] = __( 'Unable to locate needed folder (%s).' );
 
+		$this->strings['no_package']           = __( 'Update package not available.' );
```

Comment:
This key is typically set in child classes, but is called in this class. To test this class, and to make things more robust, I suggest adding this key with a default value.